scripts/operation/docker-compose.yml

```diff
@@ -101,25 +100,23 @@ services:
     environment:
       API_HOST: master-api
       API_PORT: ${API_PORT}
-      ENVIRONMENT_TYPE: TEST
-      #Other provisioning data:
-      #ENVIRONMENT_TYPE: PROD
+      PROVISIONING_TYPE: ${PROVISIONING_TYPE}
       ROOT_SECRET: ${ROOT_SECRET}
       ORGANIZATION: ${ORGANIZATION}
     depends_on:
       - master-api
 
   excel-export-service:
-    image: trubudget/excel-export:${TAG}
+    # ToDo: Replace local build with image from dockerhub. Local image must be used because of new/renamed env vars (API_HOST, API_PORT)
+    # image: trubudget/excel-export:${TAG}
+    build:
+      context: ../../excel-export-service
     environment:
-      PROD_API_HOST: master-api
-      PROD_API_PORT: ${API_PORT}
-      TEST_API_HOST: master-api
-      TEST_API_PORT: ${API_PORT}
+      API_HOST: master-api
+      API_PORT: ${API_PORT}
       PORT: ${EXPORT_PORT}
```

Comment:
This todo should be resolved after this PR is merged, because the env var names are hardcoded in the images of dockerhub.